db/migrate/20200806120434_change_node_properties_to_long_text.rb

```diff
@@ -0,0 +1,7 @@
+class ChangeNodePropertiesToLongText < ActiveRecord::Migration[5.2]
+  TEXT_BYTES = 1_073_741_823
+
+  def change
+    change_column :nodes, :properties, :text, limit: TEXT_BYTES
```

Comment:
@aapomm Just wondering... how does 1_073_741_823 get translated to `limit: 4294967295` after migration?

Reply:
From the `db/migrate/20151119124120_create_versions.rb` file:

>   The largest text column available in all supported RDBMS is
  1024^3 - 1 bytes, roughly one gibibyte.  We specify a size
  so that MySQL will use `longtext` instead of `text`.  Otherwise,
  when serializing very large objects, `text` might not be big enough.